js/EmbedTopicsRequest.js

```diff
@@ -10,34 +10,75 @@ export const SCHEMA = {
   "description": "Schema for embed topics requests received by RayServe",
   "additionalProperties": false,
   "definitions": {
-    "ancestors": {
-      "type": "array",
-      "description": "The ancestors of the topic, in order, from the parent to the root",
-      "items": {
-        "$ref": "#/definitions/topic"
-      }
+     "id": {
+      "type": "string",
+      "description": "The ID of the topic content node on Studio"
+    },
+    "title": {
+      "type": "string",
+      "description": "The title of the topic"
+    },
+    "description": {
+      "type": "string",
+      "description": "The description of the topic"
     },
     "language": {
       "type": "string",
       "description": "Language code from https://github.com/learningequality/le-utils/blob/main/le_utils/resources/languagelookup.json",
       "pattern": "^[a-z]{2,3}(?:-[a-zA-Z]+)?$"
     },
+    "level": {
+      "type": "integer",
+      "description": "The level of the ancestor, where the root is 0 and the parent is the highest level"
```

Comment:
@bjester this should be better I guess?

Reply:
The implementation of the breadcrumb logic is [here](https://github.com/learningequality/curriculum-automation/pull/3/files#diff-9cc36d0170f00e05923ae3a3ddafa96ff50894661b0c655582c51062617b6b00R14-R22) for context

Reply:
Looks great!